### DISCUSSION TITLE
Cleanup Installation

### DISCUSSION THREAD

**Discussion by bloesss** [amidaware/tacticalrmm#1612] [Q&A]
Hi,

I messed up with Firewall, external IP, Domains and Databases and the Frontend just does not show up.
I would like to cleanup the defective TRMM Installation and begin from zero.

What would I have to to? Purge Nginx, Purge Postgres, what else?

**Comment by dinger1986** [comment]
Wipe the machine and do it again.

**Reply by bloesss** [reply]
The Machine is a Ubuntu Cloudserver and hard to reinstall.

**Reply by dinger1986** [reply]
The best way to do it is to wipe the machine, you might be ok if you remove /rmm folder, /meshcentral folder, nginx configs, drop the db and remove entries from /etc/hosts and folder in /var/www. Will need to stop all services, deactivate, remove and reload systemctl. 

Basically look at the install script and reverse it

**Reply by silversword411** [reply]
> Cloudserver and hard to reinstall

Isn't that like an oxymoron? Cloud server you just redeploy and you have a new empty machine. 10 clicks on most hosting panels
